client/lib/cart-values/cart-items.js

```diff
@@ -363,7 +363,11 @@ function themeItem( themeSlug, source ) {
  * @returns {Object} the new item as `CartItemValue` object
  */
 function domainRegistration( properties ) {
-	return domainItem( properties.productSlug, properties.domain );
+	return {
+		product_slug: properties.productSlug,
+		meta: properties.domain,
+		is_domain_registration: true
+	};
```

Comment:
I think you could reduce code duplication by doing the following:

```javascript
function domainRegistration( properties ) {
  return extend( domainItem( properties.productSlug, properties.domain ), { is_domain_registration: true } );
}
```

That's the approach we used in other functions such as [`googleApps()`](https://github.com/Automattic/wp-calypso/blob/b87395b4b5221a641457c5458239a70a6c51b56c/client/lib/cart-values/cart-items.js#L413).